sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 4.0.0 (unreleased)
+## 4.0.0
 
+- GA release.
```

Comment:
Optional: I'm never sure if GA is too much Microsoft vocabulary (myself learned it here), hence I prefer talking about "stable" or something like that